docs/getting-started.md

```diff
@@ -66,22 +67,26 @@ function CharacterCount() {
 }
 ```
 
-If we put everything together
+Something to note is that a subscription on the underlying observable must be present before the hook is executed. We can use `Subscribe` to help us with it:
 
 ```tsx
 function CharacterCounter() {
   return (
     <div>
-      <TextInput />
-      <CharacterCount />
+      <Subscribe source$={charCount$}>
+        <TextInput />
+        <CharacterCount />
+      </Subscribe>
     </div>
   )
 }
 ```
 
 This is shown as:
 
-<CharacterCounter />
+<BrowserOnly>
+  {() => <CharacterCounter />}
+</BrowserOnly>
```

Comment:
Hah - Build failed with error `ReactDOMServer does not yet support Suspense.` - This example never triggered suspense because in this case the stream is synchronous, but now `Subscribe` does until it has subscribed to it.